sprint7-js-intro/ex2.js

```diff
@@ -0,0 +1,5 @@
+function tellFortune (numarCopii, numePartener, locatie, profesie) {
+    return "Vei fi un " + profesie + " in " + locatie + ", casatorit cu " + numePartener + " si vei avea " + numarCopii + " copii"
+}
+
+console.log(tellFortune(10, "Ana", "Filipine", "sofer de camion"))
```

Comment:
```suggestion
console.log(tellFortune(10, "Ana", "Filipine", "sofer de camion"));
```

se poate adauga inca un exemplu